### DISCUSSION TITLE
How can I adjust the height automatically based on the device's width?

### DISCUSSION THREAD

**Discussion by Shubhankar-1** [LonelyCpp/react-native-youtube-iframe#303] [Q&A] [answered]
How can I adjust the height automatically based on the device's width? Different devices have varying widths, and using a fixed height causes the player to be cropped. Could someone assist me with this issue?

**Comment by vandero** [comment]
I have the same problem. Any ideas?

**Reply by Shubhankar-1** [reply]
I think we need calculate height and width of device dynamically

**Comment by ArnabXD** [comment, marked as answer]
I use this to have a 16:9 aspect ratio

```
height={(9 / 16) * width}
```

**Reply by Shubhankar-1** [reply]
Yeah, I have also used the same.
